caql/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>caql</artifactId>
+	<packaging>jar</packaging>
+	<name>CaQL grammar</name>
+	<parent>
+		<groupId>org.antlr.grammars</groupId>
+		<artifactId>grammarsv4</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.antlr</groupId>
+				<artifactId>antlr4-maven-plugin</artifactId>
+				<version>${antlr.version}</version>
+				<configuration>
+					<sourceDirectory>${basedir}</sourceDirectory>
+					<includes>
+					   <include>CaQLLexer.g4</include>
+					   <include>CaQLParser.g4</include>
+					</includes>
+					<visitor>true</visitor>
+					<listener>true</listener>
+				</configuration>
+				<executions>
+					<execution>
+						<goals>
+							<goal>antlr4</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>com.khubla.antlr</groupId>
+				<artifactId>antlr4test-maven-plugin</artifactId>
+				<version>${antlr4test-maven-plugin.version}</version>
+				<configuration>
+					<verbose>false</verbose>
+					<showTree>false</showTree>
+					<entryPoint>expression</entryPoint>
+					<grammarName>CaQL</grammarName>
+					<packageName></packageName>
+					<exampleFiles>SampleData.txt</exampleFiles>
```

Comment:
This doesn't work with the Antlr test plugin. Change this to `<exampleFiles>examples/</exampleFiles>` `, then create a directory `examples` in your grammar directory and move `SampleData.txt` to it. Then, run `mvn clean test`. You should see something like `Parsing :C:\msys64\home\Kenne\caql\grammars-v4\caql\examples\SampleData.txt` in the output. Also, you'll need to change the "root pom.xml" to include the grammar because Maven won't go into the file system looking for caql/pom.xml. Insert it [here](https://github.com/antlr/grammars-v4/blob/b231b2efba53cbf60c9a9933973146f23dae9fcc/pom.xml#L97).